python-avd/pyavd/_cv/workflows/deploy_configs_to_cv.py

```diff
@@ -111,7 +111,8 @@ async def get_existing_device_container_ids_from_root_container(workspace_id: st
     LOGGER.info("deploy_configs_to_cv: Found %s root containers.", len(root_containers))
     if CONFIGLET_CONTAINER_ID not in root_containers:
         LOGGER.info("deploy_configs_to_cv: AVD root container not assigned as root container in Static Config Studio. Fixing...")
-        root_containers.append(CONFIGLET_CONTAINER_ID)
+        # Inserting our container first, to allow reconcile and other static config containers to override the AVD config.
```

Comment:
I would update the log message above to clarify this.

Reply:
The log msg should not explain the full behavior of the code and background. But I can improve the message a bit.

Reply:
updated